pkg/archive/dev_freebsd.go

```diff
@@ -0,0 +1,7 @@
+//go:build freebsd
+
+package archive
+
+import "golang.org/x/sys/unix"
+
+var mknod = unix.Mknod
```

Comment:
More a "comment" than "need to change". I took this approach as well in some cases, but found that sometimes it became confusing, because this will be listed under `variables` in GoDoc, not under `functions`. Less relevant for non-exported functions, but for that reason I sometimes started to lean towards adding a wrapper function instead, i.e..

```go
func mknod(path string, mode uint32, dev uint64)) error {
	return unix.Mknod(path, mode, dev)
}
```

(no need to change, fine as-is IMO)